brion/morphologyPlugin.h

```diff
@@ -35,10 +35,12 @@ namespace brion
 class MorphologyInitData : public PluginInitData
 {
 public:
-    explicit MorphologyInitData(const URI& uri)
+    explicit MorphologyInitData(
+        const URI& uri, const MorphologyVersion v = MORPHOLOGY_VERSION_H5_1_1,
+        const CellFamily f = FAMILY_NEURON)
         : PluginInitData(uri, MODE_READ)
-        , version(MORPHOLOGY_VERSION_H5_1_1)
-        , family(FAMILY_NEURON)
+        , version(v)
+        , family(f)
```

Comment:
This should throw when CellFamiliy is FAMILY_GLIA and the version is not 1.1. The old code was a bit strange exactly to prevent that from being possible at all.